sdk/storage/Azure.Storage.Queues/src/Models/QueueMessage.cs

```diff
@@ -35,7 +35,17 @@ internal QueueMessage(string messageText)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public string MessageText
         {
-            get => Body?.ToString();
+            get
+            {
+                try
+                {
+                    return Body?.ToString();
+                }
+                catch (ArgumentNullException) // workaround for: https://github.com/dotnet/runtime/issues/68262 which was fixed in 8.0.0, can remove this after upgrade
```

Comment:
https://github.com/Azure/azure-sdk-for-net/issues/41941#issuecomment-2371610274

Reply:
Should we add this formally to the issue description for that one and rework it so be more "upgrade to 8.0" general?

Reply:
Added a new issue and linked the related ones - https://github.com/Azure/azure-sdk-for-net/issues/46280